From d4061be4c72b952f1c20ec770f910ee311850478 Mon Sep 17 00:00:00 2001
From: Gary Li <gary.li1@uwaterloo.ca>
Date: Wed, 28 Dec 2022 22:19:57 +0000
Subject: [PATCH] nautilus-window: Fix window maximized notify signal

Nautilus does not persist the window maximized state.

This is because we are listening to the property is-maximized for
change, which is now maximized in GTK4.
https://docs.gtk.org/gtk4/property.Window.maximized.html

Use notify::maximized instead of notify::is-maximized.

Fixes: https://gitlab.gnome.org/GNOME/nautilus/-/issues/2682
---
 src/nautilus-window.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/nautilus-window.c b/src/nautilus-window.c
index 7cd980d987..092afcbbd2 100644
--- a/src/nautilus-window.c
+++ b/src/nautilus-window.c
@@ -2084,7 +2084,7 @@ nautilus_window_init (NautilusWindow *window)
                              window,
                              G_CONNECT_SWAPPED);
 
-    g_signal_connect (window, "notify::is-maximized",
+    g_signal_connect (window, "notify::maximized",
                       G_CALLBACK (on_is_maximized_changed), NULL);
 
     window->slots = NULL;
-- 
GitLab

